shortcodes/espresso_thank_you/templates/thank-you-page-transaction-details.template.php

```diff
@@ -44,7 +42,7 @@
                     <?php $transaction->e_pretty_status(true);
                     if ($show_try_pay_again_link && ! $transaction->is_completed()) { ?>
                         <span class="small-text">
-                        <a href='<?php echo $SPCO_payment_options_url ?>'>
+                        <a href='<?php echo esc_url($SPCO_payment_options_url); ?>'>
```

Comment:
`esc_url()` is fine for display, but for URLs that need to work, it's better to use `esc_url_raw()`